diff --git a/settings.ini b/settings.ini
index 691a73d6..8e9ed90a 100644
--- ./settings.ini
+++ ./settings.ini
@@ -13,7 +13,7 @@ status = 4
 min_python = 3.6
 audience = Developers
 language = English
-requirements = fastcore>=1.1.0 torchvision>=0.7 matplotlib pandas requests pyyaml fastprogress>=0.2.4 pillow scikit-learn scipy spacy
+requirements = fastcore>=1.1.0 torchvision>=0.7 matplotlib pandas requests pyyaml fastprogress>=0.2.4 pillow scikit-learn scipy spacy graphviz
 pip_requirements = torch>=1.6.0
 conda_requirements = pytorch>=1.6.0
 dev_requirements = nbdev>=1.0.10 ipywidgets pytorch-lightning pytorch-ignite transformers catalyst sentencepiece<0.1.90 tensorboard pydicom catalyst captum
