diff --git a/settings.ini b/settings.ini
index 23bdebe6..9243a183 100644
--- ./settings.ini
+++ ./settings.ini
@@ -13,7 +13,7 @@ status = 4
 min_python = 3.6
 audience = Developers
 language = English
-requirements = fastcore>=1.3.8 torchvision>=0.8 matplotlib pandas requests pyyaml fastprogress>=0.2.4 pillow scikit-learn scipy spacy
+requirements = fastcore>=1.3.8 torchvision>=0.8 matplotlib pandas requests pyyaml fastprogress>=0.2.4 pillow scikit-learn scipy spacy graphviz
 pip_requirements = torch>=1.7.0
 conda_requirements = pytorch>=1.7.0
 dev_requirements = nbdev>=1.0.10 ipywidgets pytorch-lightning pytorch-ignite transformers catalyst sentencepiece tensorboard pydicom catalyst captum>=0.3 flask neptune-client wandb kornia scikit-image
